.gentoo/sci-biology/mouse-brain-atlases_generator/mouse-brain-atlases_generator-9999.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Virtual : Generate a package-manager friendly mouse brain atlases collection"
+
+SLOT="0"
+KEYWORDS=""
+
+RDEPEND="
+	dev-python/numpy
+	sci-biology/fsl
+	sci-libs/nibabel
+	sci-biology/ants
+	sci-biology/nilearn
+	sci-libs/scikits_image
+	media-gfx/blender
+	dev-python/pynrrd
+	"
```

Comment:
Please add an `elog` message informing the user that this is an experimental package which is designed only to handle dependencies, and that no files get installed and scripts have to be executed manually inside the repostitory.

You can find examples with `qgrep elog`.